### DISCUSSION TITLE
Move the selection when window is scrolled

### DISCUSSION THREAD

**Discussion by benwoodward** [infokiller/web-search-navigator#396] [General]
Great extension!

I have a feature request. I use Vimium as well as Web Search Navigator, this is a situation I keep running into:

• Invoke the Web Search Navigator shortcut to move the search result selection down to the next result
• Realise there isn’t anything in the top 5 results so decide to scroll down the page and scan the remaining results
• Instead of hammering Web Search Navigator shortcut to move to next result, I either scroll with trackpad or Vimium shortcut.
• The search result selection then moves off the top of window
• If I find something to click at the bottom of the search results, I either have to use Vimium’s link clicking shortcut, or hammer the Web Search Navigator shortcut 20 times to move the selection down to where I am
• To avoid this I have developed a habit of Vimium page down shortcut  … hammer Web Search Navigator shortcut 5 times … Vimium page down shortcut … hammer Web Search Navigator shortcut 5 times … repeat

It’d be great if the search selection automatically moved to the first visible result if the window is scrolled to prevent the above scenario.

**Comment by infokiller** [comment]
@benwoodward do you still use the extension, and if so are you still interested in this feature?
I also use both this extension and Vimium (actually https://github.com/gdh1995/vimium-c). I never run into this issue because I only "scroll" with this extension's shortcuts in Google (but I do use Vimium navigation commands elsewhere). Have you tried this?
I'm open to merging a PR adding an option to move the selection when the window is scrolled, but I will probably not get to implementing it myself.

**Reply by benwoodward** [reply]
Hi there, I stopped using this extension in favour of Vimium + Vimac a while ago. Thanks
